test/correctness/pytorch.cpp

```diff
@@ -149,11 +148,21 @@ inline int test1_th_(float _alpha, int32_t _beta, at::Tensor &_buf) {
     }
 
     {
+        // No: parsing a target string with 'cuda' in it will always call `calculate_host_cuda_capability()`,
```

Comment:
If this is right, it's a bug. It's supposed to only call calculate_host_cuda_capability if the target string contains both "host" and "cuda". The target string I suggested earlier shouldn't call that function.

"host" means "please sniff my host system".

Reply:
The conditions under which we call it are:
```
is_host &&
        t.has_feature(Target::CUDA) &&
        !t.has_feature(Target::CUDACapability30) &&
        !t.has_feature(Target::CUDACapability32) &&
        !t.has_feature(Target::CUDACapability35) &&
        !t.has_feature(Target::CUDACapability50) &&
        !t.has_feature(Target::CUDACapability61) &&
        !t.has_feature(Target::CUDACapability70) &&
        !t.has_feature(Target::CUDACapability75) &&
        !t.has_feature(Target::CUDACapability80))
```

So neither "linux-x86-64-cuda" nor "host-cuda-cuda_capability_30" will call that function.